fMRIVolume/scripts/OneStepResampling.sh

```diff
@@ -30,17 +30,18 @@ Usage() {
   echo "             --scoutgdcin=<input scout gradient nonlinearity distortion corrected image (EPI pre-sat)>"
   echo "             --oscout=<output transformed + distortion corrected scout image>"
   echo "             --ojacobian=<output transformed + distortion corrected Jacobian image>"
+  echo "             --refreg=<whether to do 'linear', 'nonlinear' or no ('NONE') additional registration to movement reference image>"
```

Comment:
The framing of this usage statement, with 3 values gives the impression that 'linear' is actually doing something.  But it appears that 'linear' and 'NONE' are the same.  Thence my suggestion that we consider using `--refregnonlinear=(TRUE|FALSE)` instead.

Reply:
No, NONE is used in the case when there is no external reference to register to. This is linked to PR <https://github.com/Washington-University/HCPpipelines/pull/140>